hpedockerplugin/volume_manager.py

```diff
@@ -179,27 +242,20 @@ def create_snapshot(self, src_vol_name, snapshot_name,
                     'retentionHours': retention_hrs,
                     'display_description': 'snapshot of volume %s'
                                            % src_vol_name}
-        try:
-            bkend_snap_name = self._hpeplugin_driver.create_snapshot(snapshot)
-            undo_steps.append(
-                {'undo_func': self._hpeplugin_driver.delete_volume,
-                 'params': {'volume': snapshot,
-                            'is_snapshot': True},
-                 'msg': 'Cleaning up backend snapshot: %s...'
-                        % bkend_snap_name})
-        except Exception as ex:
-            msg = (_('create snapshot failed, error is: %s'),
-                   six.text_type(ex))
-            LOG.error(msg)
-            return json.dumps({u"Err": six.text_type(ex)})
-
-        response = json.dumps({u"Err": ''})
         db_snapshot = {'name': snapshot_name,
                        'id': snapshot_id,
+                       'parent_name': src_vol_name,
                        'parent_id': vol['id'],
                        'expiration_hours': expiration_hrs,
                        'retention_hours': retention_hrs}
         vol['snapshots'].append(db_snapshot)
+
```

Comment:
Parent volume is holding irrelevant information in terns of expiration and retention hours